test/system/220-healthcheck.bats

```diff
@@ -273,4 +273,215 @@ Log[-1].Output   | \"Uh-oh on stdout!\\\nUh-oh on stderr!\\\n\"
     done
 }
 
+@test "podman healthcheck --health-max-log-count default value (5)" {
+    local repeat_count=10
+    local msg="Hello, How are you?"
+    local ctrname="c-h-$(safename)"
+    run_podman run -d --name $ctrname   \
+               --health-cmd "echo $msg" \
+               $IMAGE /home/podman/pause
+    cid="$output"
+
+    run_podman inspect $ctrname --format "{{.Config.HealthMaxLogCount}}"
+    is "$output" "5" "HealthMaxLogCount is set to 5"
```

Comment:
Please drop the "is set to 5": on failure, the diagnostic message will already say "expected 5, got (something else)". And if the expected value ever changes, someone might change the "5" but forget to change the "is set to 5", and the code will be impossible to understand.

Also, where does this magic 5 come from? Is it a hardcoded default in podman? Is it a containers.conf setting? (I could read the PR but this is way too huge right now). What will happen if/when that default changes or is overridden? Maybe the message should say "HealthMaxLogCount is the expected default", or some explanation of where it's coming from